locustfiles/prime_workflow.py

```diff
@@ -3,19 +3,13 @@
 
 from tasks.prime_endpoint_workflows import PrimeEndpointWorkflowsTasks
 from tasks.prime_hhg_workflow import PrimeWorkflowTasks
-from utils.hosts import MilMoveDomain, MilMoveHostMixin
 
 
-class PrimeWorkflowUser(MilMoveHostMixin, HttpUser):
+class PrimeWorkflowUser(HttpUser):
     """This is the Workflow user that will be calling the workflows in prime_workflow.py.
     This user needs access to both prime and support apis.
     """
 
-    local_port = "9443"
-    domain = MilMoveDomain.PRIME  # the base domain for the host
-
-    wait_time = between(0.25, 9)  # the time period to wait in between tasks (in seconds, accepts decimals and 0)
-    weight = 1
-
-    # For the tasks, we can define all the workflow tasksets we have set up:
     tasks = {PrimeWorkflowTasks: 1, PrimeEndpointWorkflowsTasks: 1}
+    wait_time = between(0.25, 9)
+    weight = 1
```

Comment:
This isn't new to your PR but I just noticed this has a weight when the other User locustfiles don't specify one.  Any idea if we need this?

Reply:
Oh right, I forgot to look into it when I was making the conversions. I'm not sure, I'll check.

Reply:
Ah apparently it does have a use, but in this case we're using the default so we could get rid of it: https://docs.locust.io/en/stable/api.html#locust.User.weight

Reply:
Thanks, yea it didn't seem different than the default.